### DISCUSSION TITLE
Welcome to TRN-EmptyR Team

### DISCUSSION THREAD

**Discussion by ery-day-bradly** [ery-day-bradly/TRN-EmptyR#3] [Announcements]
## 👋 Welcome Team!
   Hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together _💪ibid throughout._

  To get started, comment below with an introduction of yourself and tell us about what you do with this team.

**Comment by HarrisMBrad-cell** [comment]
Are we: prepped and ready to lift off? 
 [ #21 `Ready To 🚀 and start dev`
 - Awe Yeah... #22]
